apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts

```diff
@@ -12,7 +12,8 @@ const getBaseUrl = (): string => {
   if (isDev) return 'https://dev.swap.cow.fi'
   if (isVercel) {
     const prKey = window.location.hostname.replace('widget-configurator-git-', '').replace('-cowswap.vercel.app', '')
-    return `https://swap-dev-git-${prKey}-cowswap.vercel.app`
+
+    return `https://swap-dev-git-${prKey}-cowswap-dev.vercel.app`
```

Comment:
This project is still under `cowwap` (prod) team on Vercel

![image](https://github.com/user-attachments/assets/51e4e2e1-698b-4ebe-88dd-b91b1e3e313a)


@shoom3301